src/Tickets/Commerce/Gateways/Stripe/WhoDat.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\Stripe;
+
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_WhoDat;
+use TEC\Tickets\Commerce\Gateways\Stripe\REST\On_Boarding_Endpoint;
+
+/**
+ * Class WhoDat. Handles connection to Stripe when the platform keys are needed
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Stripe
+ */
+class WhoDat extends Abstract_WhoDat {
+
+	/**
+	 * The API URL.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+//	protected $api_url = 'https://whodat.theeventscalendar.com/commerce/v1/stripe';
+	protected $api_url = 'https://localhost/whodat/index.php/commerce/v1/stripe';
```

Comment:
We should modify this on the Abstract so we can change with a variable on your WP Config.